src/main/java/com/conveyal/analysis/controllers/OpportunityDatasetController.java

```diff
@@ -148,6 +148,8 @@ private OpportunityDatasetUploadStatus downloadLODES(Request req, Response res)
         }
 
         final String regionId = req.params("regionId");
+        final int zoom = req.params("zoom") != null ? Integer.parseInt(req.params("zoom")) : ZOOM;
```

Comment:
@trevorgerhardt what do you think about this approach? 

Zoom level should be adjustable for:
* analysis (previously implemented)
* LODES import (here)
* spatial dataset and aggregation area upload (soon)
* prebuilt and saved linkages/egress cost tables (longer-term)

One approach would be setting the zoom once at the region level, then reading it from the region model object in the database. 

Alternatively, the front-end could include a zoom selector for each of the uses above (maybe defaulting to a value set at the region level, but allowing customization).